README.md

```diff
@@ -89,5 +89,5 @@ Use Docker to build and serve the content, but remember to rebuild the Docker im
 
 [Contributors](contributors.md)
 
-![](img/class_light.svg ':size=100x100 :class=light-mode-icon' alt="image of class; light mode")
-![](img/class_dark.svg ':size=100x100 :class=dark-mode-icon' alt="image of class; dark mode")
+![classroom image](img/class_light.svg ':size=100x100 :class=light-mode-icon :alt= classroom image; light mode')
+![classroom image](img/class_dark.svg ':size=100x100 :class=dark-mode-icon :alt= classroom image; dark mode')
```

Comment:
```suggestion
![classroom image](img/class_light.svg ':size=100x100 :class=light-mode-icon :alt=classroom image; light mode')
![classroom image](img/class_dark.svg ':size=100x100 :class=dark-mode-icon :alt=classroom image; dark mode')
```

maybe a NP. not sure if spaces matter here

Reply:
yah, that'll break it 😢

Reply:
rather, you'll lose the word `classroom` for your definition

Reply:
Good to know thanks for the info